api/millenium_patron.py

```diff
@@ -99,6 +106,31 @@ class MilleniumPatronAPI(BasicAuthenticationProvider, XMLParser):
           "default": PIN_AUTHENTICATION_MODE
         }
     ] + BasicAuthenticationProvider.SETTINGS
+
+    LIBRARY_SETTINGS = [
+        { "key": PATRON_FIELD_RESTRICTION_FIELD,
+          "label": _("Patron record restriction field"),
+          "type": "select",
+          "description": _("Field on the patron record used for <i>patron record restriction</i> below."),
+          "options": [
+              { "key": AGENCY_FIELD, "label": _(AGENCY_FIELD) },
```

Comment:
The `_` is from a localization library and part of the localization process is searching the code and pulling out strings that need to be translated, but I don't think it will work if you pass in a variable like this. Since this is a field in Sierra I don't think it should be translated anyway, so I would take out the `_`s here.